source/docs/3/basic_filtering_and_styling_with_expressions.rst

```diff
@@ -0,0 +1,120 @@
+
+  .. image:: /static/3/basic_filtering_and_styling_with_expressions/images/1.png
+    :align: center
+
+2. A new layer ``ne_50m_admin_0_countries`` will now be loaded in QGIS and you should see a map of the world. Click on :guilabel:`Open Layer Styling Panel`.
+
+  .. image:: /static/3/basic_filtering_and_styling_with_expressions/images/2.png
+    :align: center
+    
+3. Switch from :guilabel:`Single Symbol` to :guilabel:`Rule-based`.
+
+  .. image:: /static/3/basic_filtering_and_styling_with_expressions/images/3.png
+    :align: center
+    
+4. Double-click on the default rule to enter the :guilabel:`Edit Rule` dialog..
+
+  .. image:: /static/3/basic_filtering_and_styling_with_expressions/images/4.png
+    :align: center
+
+5. To filter using an expression, Click the Filter :guilabel:`ε`. 
+
+  .. image:: /static/3/basic_filtering_and_styling_with_expressions/images/5.png
+    :align: center
+
+6. You have entered the  :guilabel:`Expression Builder`. This dialog provides the main interface to write expressions and can be accessed in many ways. It contains an input box on the left, a list of functions in the middle and a help box on the right.
+
+  .. image:: /static/3/basic_filtering_and_styling_with_expressions/images/6.png
+    :align: center
+
+7. From the list in the middle, expand :menuselection:`Fields and Values` to explore the fields found in the ``ne_50m_admin_0_countries`` layer. 
+
+  .. image:: /static/3/basic_filtering_and_styling_with_expressions/images/7.png
+    :align: center
+
+8. Click to select ``SUBREGION`` field and click :guilabel:`All Unique` on the right panel to view all unique subregion values. 
+
+  .. image:: /static/3/basic_filtering_and_styling_with_expressions/images/8.png
+    :align: center
+
+9. After exploring, select the :guilabel:`Expression` box on the left to input the expression as follows and click :guilabel:`OK`.
+
+.. code-block:: none
+
+  "SUBREGION" = 'South-Eastern Asia'
+
+.. image:: /static/3/basic_filtering_and_styling_with_expressions/images/9.png
+    :align: center
+  
+10. You should see SE Asia. Pan and zoom to the center to the region in your map window.
+
+  .. image:: /static/3/basic_filtering_and_styling_with_expressions/images/10.png
+    :align: center
+
+11. Now that we have used an expression to filter features, let us use an expression to color each country based on a field ``MAPCOLOR7``. Scroll-down to the :guilabel:`Symbol` options in the  styling panel and select :guilabel:`Simple Fill` to bring up styling options such as fill color, line color and more. Click on :guilabel:`Data define override` button for the :guilabel:`Fill color` and select :guilabel:`Edit...` on the menu.
+
+  .. image:: /static/3/basic_filtering_and_styling_with_expressions/images/11.png
+    :align: center
+	
+12. This brings up the :guilabel:`Expression Builder` dialog for Fill color. Enter the following expression:
+
+.. code-block:: none
```

Comment:
indent the start of the code-clock to match the indent level of the image. Fix other instances also